tests/python_tests/test_llm_pipeline.py

```diff
@@ -812,14 +812,26 @@ def py_streamer(py_str: str):
 
 @pytest.mark.parametrize("pipeline_type", get_gguf_pipeline_types())
 @pytest.mark.parametrize("model_ids", get_gguf_model_list())
+<<<<<<< HEAD
```

Comment:
please, resolve conflicts once https://github.com/openvinotoolkit/openvino.genai/pull/2090 is merged